docs/syntax/html.md

```diff
@@ -68,9 +68,22 @@ Markup may contain elements that do not work without accompanying JavaScript.  T
 	<div class="o--if-js">Submit a new comment: ... </div>
 	<div class="o--if-no-js">To comment on this article, you need to upgrade your web browser.  <a href="...">Learn how to upgrade</a>.</div>
 
-To avoid unnecessary HTTP requests, elements with the class `o--if-no-js` *must not* be (or contain) `<img>` tags, and *must not* have a background image URL set with CSS.  Descendent elements of the `o--if-no-js` element *may* have CSS image backgrounds ([Learn more](http://timkadlec.com/2012/04/media-query-asset-downloading-results/))
-
-* Learn more about [Core vs enhanced experience]({{site.baseurl}}/docs/developer-guide/using-modules/#core-vs-enhanced-experience)
+With this method it is possible to trigger unwanted HTTP requests to be made should you include an image within the `o--if-no-js` element or use a CSS background image on the element ([Learn more](http://timkadlec.com/2012/04/media-query-asset-downloading-results/)). If you require an image to be added to the page for core experience only, you could use a `<noscript>` tag:
```

Comment:
How about rewording to this:


Elements with the class `o--if-no-js` will trigger unnecessary HTTP requests if they are (or contain) `<img>` tags or have  a background image URL set with CSS [Learn more](http://timkadlec.com/2012/04/media-query-asset-downloading-results/). 
If you require an image to be added to a page for core experience only, you *should* use a `<noscript>` tag: